### ISSUE TITLE
Zettlr slows down mac (nightly)

### ISSUE THREAD

**Issue by halloichbingunnar** [Zettlr/Zettlr#3906]
### Description

When having Zettlr open, stage manager or other visual operations are considerably lagging. The stable release does not influence the performance in any way.

### Reproducing

Open Zettlr 2.3.0 :)

### Zettlr Version

Nightly

### Specify version

2.3.0

### Installation Method

From the Website or GitHub

### Your Platform

- [ ] Windows
- [X] macOS
- [ ] Linux

### Architecture

- [X] Intel 64bit
- [ ] ARM 64bit

### Operating System Version

Ventura 13.0

**Comment by nathanlesage** [comment]
The same should, honestly, also happen on 2.3 as the main cause for the lag is the fact that the window is shine-through (it has this "vibrancy" thing set so that things behind the window influence the color of the left and right sidebar). I don't think there should be any difference between Ventura and macOS 12. If there are, I'm even willing to assume that this might also be an interaction between Stage Manager (which is not available on macOS 12).

I'll have to implement a switch to turn off vibrancy before we can validate this behavior, that could take some time …

**Comment by nathanlesage** [comment]
@halloichbingunnar Did performance improve in the current beta?

**Comment by halloichbingunnar** [comment]
it might have improved a little, but it is still lagging in comparison to switching between other apps…

**Comment by nathanlesage** [comment]
Ah, wait a moment. I re-read your issue – you're complaining that when you move the window around (or if macOS moves the window around) it's laggy? That's because I'm using a translucent background (which puts Zettlr more in line with other macOS apps). It's normally unnoticeable, unless one moves the whole window around a lot, e.g., by using stage manager. Can you confirm that? If so, I meant to implement a switch to enable or disable that behavior at some point.

**Comment by halloichbingunnar** [comment]
yes, that's the case. I haven't noticed it for any other app, but I will look out for it.

**Comment by halloichbingunnar** [comment]
but actually, now that I think about it again, it must have improved between the versions – there is a lagging at the moment, but I'm not sure if it would have made me open up an issue to complain about it…

**Comment by nathanlesage** [comment]
Does this mean there is still lagging or not …? :D

**Comment by halloichbingunnar** [comment]
Haha, let's say "No" and close the issue ;)
